CHANGELOG.md

```diff
@@ -24,6 +24,12 @@
 * `[docs]` Add documentation for .toHaveProperty matcher to accept the keyPath
   argument as an array of properties/indices.
   ([#5220](https://github.com/facebook/jest/pull/5220))
+* `[jest-runner]` test environments are now passed a new `options` parameter.
```

Comment:
I moved these to the `Features` section because whether this is a bug or a new feature is debatable, so it made more sense as a feature to me.